### PR TITLE
Nav bar

### LINKED ISSUE #46: Logged in user can log out
As a logged in user, when I visit any page, I can log out.

- [ ] Happy/sad path testing

- [ ] Update to README (FE screenshots, BE API endpoint documented)

### LINKED ISSUE #47: Nav Bar for all pages
As a user, when I visit any page, there is a navigation bar at the top of the page.

On the left is a logo that is clickable and takes you to the home page.

On the right side is the button to either login with google or logout

- [ ] Happy/sad path testing

- [ ] Update to README (FE screenshots, BE API endpoint documented)

### PR DESCRIPTION
**Pull request by nmking22**
This PR Will:
* Add user logout functionality
* Add flash messages for logging in/out
* Add navbar

- [x] Routes Added: get '/logout', to: 'sessions#destroy', as: :google_logout

- [x] Repo Representation: Front End

- [x] Sad Path Tested - sad paths handled with google auth

- [x] Happy Path Tested

- [x] All Tests are passing

- [x] Collaborators: Sheryl Stillman

Closes #47 
Closes #46

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -10,6 +10,15 @@
   </head>
 
   <body>
+    <nav class="navbar navbar-default" role="navigation">
+      <div class="navbar-header">
+        <% if !current_user %>
+          <%= link_to "Log In with Google", google_login_path, class: "navbar-brand" %>
```
Comment (stillsheryl) [inline app/views/layouts/application.html.erb:16]: We figured we can use styling later to have a logo on the left for the home button, and the login/out on the right-hand side.

**Review by stillsheryl — APPROVED** [review]

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (iandouglas) [reply app/views/layouts/application.html.erb:16]: in Ruby, `if !` should be replaced with `unless` as a standard preference.

Comment (nmking22) [reply app/views/layouts/application.html.erb:16]: Thanks for the insight, Ian. We just changed the conditional to `unless`.

**Review by cpfergus1 — APPROVED** [review]
Thanks for changing if ! to unless!

**Review by dunlapww — APPROVED** [review]
Nice work!

**Review by Todd-Estes — APPROVED** [review]
Nice work yall!

**Pull request merged**